### DISCUSSION TITLE
Building the ultimate tech team

### DISCUSSION THREAD

**Discussion by nathfavour** [community/community#61438] [Programming Help]
### Body
Hi everyone!😀

I am Nathaniel, a freelance programmer. I am a member of two open source development github organizations, bAInaryglobe and Blockchain01. 

I think github is an awesome place to build the best tech team💯

If you are also a freelance open source dev, have great ideas and looking forward to collaborate, let's discuss how to build the team together!🤓

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nathfavour ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information. I wanted to call out that `Posting or seeking job listings or contests` is not allowed, but if you're looking for folks to work on project(s) with that remains within our guidelines.

**Reply by nathfavour** [reply]
Alright, thanks🙂

**Comment by shephinphilip** [comment]
I am a freelancer in machine learning  open  source contributor .

**Reply by nathfavour** [reply]
coool, just followed ya😉

**Comment by OSEight** [comment]
I code for free! I'd love to contribute to a project.

**Reply by nathfavour** [reply]
nice, @OSEight. I just followed you💯

**Reply by OSEight** [reply]
Thanks! I'll follow back!
